Brisk iOS/Radar/RadarViewModel.swift

```diff
@@ -22,7 +22,7 @@ extension RadarViewModel {
 
 	init(_ radar: Radar) {
 		product = radar.product
-		area = radar.area ?? Area.areas(for: product).first!
+		area = radar.area ?? Area.areas(for: product).first
```

Comment:
Since `area` is optional, we can omit the nil coalescing completely. 

So just `area = radar.area` should suffice.